perf/Project.toml

```diff
@@ -51,10 +51,9 @@ UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 ArgParse = "1.1"
 ArtifactWrappers = "0.2"
 AtmosphericProfilesLibrary = "0.1"
-ClimaAtmos = "0.15.1"
 CLIMAParameters = "0.4, 0.5, 0.6, 0.7"
-ClimaComms = "0.3, 0.4"
-ClimaCore = "0.10.40"
+ClimaComms = "0.4, 0.5"
+ClimaCore = "0.10"
```

Comment:
Same comment as above - we don't have to specify the compats here since they're in the top-level Project

Reply:
Okay I removed them